packages/azure-services/src/azure-batch/pool-load-generator.spec.ts

```diff
@@ -45,16 +45,19 @@ describe(PoolLoadGenerator, () => {
             },
         };
         const expectedPoolLoadSnapshot = {
+            isIdle: false,
             tasksIncrementCountPerInterval: 60,
             targetActiveToRunningTasksRatio: 2,
             configuredMaxTasksPerPool: poolMetricsInfo.maxTasksPerPool,
+            targetMaxTasksPerPool: 64,
```

Comment:
if targetMaxTasksPerPool is being used for debugging purpose only ( to store in app insights), it is still misleading, why we even have this? we are logging the maxTasksPerPool config & tasksIncrementCountPerInterval(amount of tasks that can be added in next interval).

Reply:
tasksIncrementCountPerInterval is dynamic delta increment value where is targetMaxTasksPerPool is reflecting overall capacity/throughput of the pool currently running. There is no any issue/harm with having this values in result object as it is logged in app insight and may help do debugging/investigation